### PR TITLE
Mock unit test interactive dashboard behavior

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: R/model_dashboard.R [hunk 1]
```diff
@@ -105,3 +105,8 @@ model_dashboard <- function(model,
     utils::browseURL(report_path)
   }
 }
+
+# necessary for writing a mock test (see `/tests/testthat/test-model-dashboard.R`)
+# cf. https://testthat.r-lib.org/reference/local_mocked_bindings.html#namespaced-calls
+browse_url_wrapper <- function(...) utils::browseURL(...)
+browse_mock_function <- function() browse_url_wrapper()
```
Comment (etiennebacher) [inline R/model_dashboard.R:112]: I'm not familiar with mocking but it looks like the test passes when I use `browse_url_wrapper()`, making `browse_mock_function()` useless

**Comment by IndrajeetPatil** [comment]
@etiennebacher Thanks for looking at this closely.  

I was trying to avoid gaining another dependency in the form of _mockery_ and instead using the new experimental mocking functions in _testthat_ but its API is not at all intuitive to me, at least not as clear as the _mockery_ one. 

Mocking is working as expected now.

**Review by etiennebacher — APPROVED** [review]
LGTM (but again no experience with `mockery`), thanks

**Comment by IndrajeetPatil** [comment]
Let me make a presentation. 😅

**Pull request merged**